runtime/compiler/env/J9PersistentInfo.hpp

```diff
@@ -431,6 +433,7 @@ class PersistentInfo : public OMR::PersistentInfoConnector
    uint32_t    _JITServerPort;
    uint32_t    _socketTimeoutMs; // timeout for communication sockets used in out-of-process JIT compilation
    uint64_t    _clientUID;
+   uint64_t    _serverUID;
```

Comment:
Let's add a comment saying that, at the client, _serverUID represents the UID of the server the client is connected to.

Reply:
```suggestion
   uint64_t    _serverUID; // At the client, this represents the UID of the server the client is connected to
```